src/chrome/content/rules/Spreadshirt.xml

```diff
@@ -78,12 +78,12 @@
 	<test url="http://www.spreadshirt.se/" />
 	<test url="http://wikileaks.spreadshirt.se/" />
 
-	<securecookie host="\w+\.spreadshirt\.(at|be|ch|co\.uk|com|com\.au|de|dk|es|fi|fr|ie|it|net|nl|no|pl|se)$" name=".+" />
-
-	<exclusion pattern="^http://(www\.)?spreadshirt\.(at|be|ch|co\.uk|com|com\.au|de|dk|es|fi|fr|ie|it|net|nl|no|pl|se)/" />
+	<securecookie host=".+" name=".+" />
 
 	<rule from="^http://(\w+)\.spreadshirt\.(at|be|ch|co\.uk|com|com\.au|de|dk|es|fi|fr|ie|it|net|nl|no|pl|se)/"
```

Comment:
This rule is unneeded since there is already a catch-all rule doing the same after it.